tests/gnrc_dhcpv6_client/tests/01-run.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2018 Freie Universität Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import sys
+from testrunner import run
+
+
+def testfunc(child):
+    child.expect(r"Iface\s+\d+")
+    child.expect(r"inet6 addr:\sfe80:[0-9a-f:]+\s+scope: link")
+    child.expect(r"Iface\s+\d+")
+    child.expect(r"inet6 addr:\s+fe80:[0-9a-f:]+\s+scope: link")
+    child.expect(r"inet6 addr:\s+(?P<global_addr>[0-9a-f:]+)\s+scope: global")
+    global_addr = child.match.group("global_addr")
+    child.expect(r"(?P<global_pfx>[0-9a-f:]+)/64\s+dev #\d\s+"
+                 r"expires (?P<valid_ltime>\d+)sec\s+"
+                 r"deprecates (?P<pref_ltime>\d+)sec")
+    global_pfx = child.match.group("global_pfx")
+    assert int(child.match.group("valid_ltime")) < 4000
+    assert int(child.match.group("pref_ltime")) < 3000
```

Comment:
Because, apparently, that is what I configured KEA to advertise

https://github.com/RIOT-OS/RIOT/blob/c9bf7d023236b47bd96b659c991c867455a477ad/tests/gnrc_dhcpv6_client/kea-dhcp6.conf#L18-L19

Since past me was notoriously bad at documenting what she did and why, I have to re-read the RFC if there was a reason to do this. :-/

Reply:
I guess it doesn't really make sense to check this after all, except for checking if the supplied lifetimes are configured. However, if you use different tooling and configurations this doesn't make much sense. I'll remove those checks when squashing.